spec/models/user_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+describe User, type: :model do
```

Comment:
create Capybara spec
and test sign up - https://github.com/teamcapybara/capybara#using-capybara-with-rspec using it